src/DssSpell.sol

```diff
@@ -26,8 +26,10 @@ contract DssSpellAction is DssAction {
     string public constant override description =
         "2023-06-16 MakerDAO Executive Spell | Hash: 0xecdc5a5a1be69f23d299235ad966c82c7e03365e45aaed69ed1eb2f03daff4a6";
 
-    // Comma-separated list of DAO resolutions.
-    string public constant resolutions = "QmYPbYoPjrSzBu5gt9tip78siG1gFvY8K4HTkHEFJgMmM8";
+    // ----- 2023-06-16 DAO Resolutions -----
+    // Forum: https://forum.makerdao.com/t/out-of-schedule-executive-to-fortify-elasticity-of-on-chain-1-1-liquidity-of-usdc/21168
+    // Comma-separated list of DAO resolutions IPFS hashes.
+    string public constant dao_resolutions = "QmYPbYoPjrSzBu5gt9tip78siG1gFvY8K4HTkHEFJgMmM8";
```

Comment:
Nit: If it is a single file (per my other comment) it would be `dao_resolution`

Reply:
Won't change.
Explained the reason above: if there's need for multiple resolutions, we are already covered keeping the name in the plural.

Reply:
Non-functional and probably not easy to uphold as precedent given it made no sense when it shipped (`QmYPbYoPjrSzBu5gt9tip78siG1gFvY8K4HTkHEFJgMmM8` isn't plural) so I am not going to object to this, I was just raising the point that it made no sense when I read this (describes plural when it's singular).
Resolving